src/components/ec-smart-table/__snapshots__/ec-smart-table.spec.js.snap

```diff
@@ -103,7 +103,9 @@ exports[`EcSmartTable should render resolved data properly 1`] = `
     class="ec-loading__content ec-loading__content--is-transparent"
     data-test="ec-loading__content"
   >
-    <div>
+    <div
+      defaultsortcycle=",asc,desc"
```

Comment:
This is not a valid HTML attribute, you have to stop passing the defaultSortCycle from withSorting to the wrapped component. You can do that in the `createHOCc` call in withSorting:
```
// this bit determines what properties are being passed to the wrapped component. By default, all props coming to the withSorting are passed down to the wrapped component. On top of that, you can add more props, e.g. sorts will be passed like this:
props: {
  sorts() {
      return this.internalSorts;
    },
}
```
^^ to fix this you have to do this. change the props to a function:
```
props(props) {
      return { 
        ...props, // pass everything you've got
        sorts: this.internalSorts, // pass additional prop
        defaultSortCycle: null, // don't pass defaultSortCycle to the wrapped component
      };
    },
```

Reply:
Will do thanks